test/test.js

```diff
@@ -695,16 +696,18 @@ describe('useref()', function() {
         stream.end();
     });
 
-    it('should support external streams', function(done) {
+    it.only('should support external streams', function(done) {
         var extStream1 = gulp.src('test/fixtures/scripts/that.js')
+            .pipe(through.obj(function (newFile, enc, callback) {
+                setTimeout(callback, 75);
```

Comment:
The reason this fails is because the file is never getting sent through. Changing this line to `setTimeout(callback(null, newFile), 75);` will make it pass.